ext/nnstreamer/registerer/grpc.c

```diff
@@ -0,0 +1,61 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * nnstreamer registerer for gRPC plugin
+ * Copyright (C) 2020 Dongju Chae <dongju.chae@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
```

Comment:
You may remove these header comments. IMO, the SPDX identifier above is enough.

Reply:
Got it.